devlab/provision/generate_config.sh

```diff
@@ -34,9 +34,7 @@ while read key value
 do
     value=($value)
     value=${value[1]}
-    if [[ -n ${value} ]]; then
-      sed -i "s|<${key}>|${value}|g" ${result_config}
-    fi
+    sed -i "s|<${key}>|${value}|g" ${result_config}
```

Comment:
As I understand default value should be set when user's value is not set. So, when user keep a blank key - it means that there is no value and need to use default value to prevent any mistakes. Let's see comments from other reviewers.